save-sandbox/src/main/kotlin/com/saveourtool/save/sandbox/controller/SandboxController.kt

```diff
@@ -217,7 +218,7 @@ class SandboxController(
         .flatMap { key ->
             storage.overwrite(
                 key = key,
-                content = Flux.just(ByteBuffer.wrap(content.toByteArray()))
+                content = Flux.just(ByteBuffer.wrap(content.replace("\r\n", "\n").toByteArray()))
```

Comment:
What about macOS and its `\r`?

Reply:
@Cheshiriks The point is that the file is sent from some PC to our sandbox, which is LF. We need to convert all possible eol symbols (CRLF, LF, CR) to LF

Reply:
> System.lineSeparator() ?

backend is ran on ubuntu - System.lineSeparator - is always /n :)

Reply:
changed to `\r\n?` -- it should handle Win and Mac